tools/src/main/java/com/nedap/archie/json/JSONSchemaCreator.java

```diff
@@ -208,8 +215,17 @@ private JsonObjectBuilder createPropertyDef(BmmType type) {
     private JsonObjectBuilder createPolymorphicReference(BmmClass type) {
 
         List<String> descendants = getAllNonAbstractDescendants( type);
+        boolean addConcreteElse = false;
         if(!type.isAbstract()) {
             descendants.add(BmmDefinitions.typeNameToClassKey(type.getName()));
+            addConcreteElse = true;
```

Comment:
I'm not sure what this means and does, maybe some comments would help

Reply:
comment added. Could you review again?